engine/filters.go

```diff
@@ -77,52 +77,55 @@ func (fS *FilterS) Pass(tenant string, filterIDs []string,
 	return
 }
 
-//verifyPrefixes verify the Element and the Values if has as prefix one of the prefixes
+//checkPrefix verify if the value has as prefix one of the prefixes
+func checkPrefix(value string, prefixes []string) (hasPrefix bool) {
+	for _, prefix := range prefixes {
+		if strings.HasPrefix(value, prefix) {
+			hasPrefix = true
+			break
+		}
+	}
+	if !hasPrefix {
+		return false
+	}
+	return
+}
+
+//verifyPrefixes verify the Element and the Values from FilterRule if has as prefix one of the prefixes
 func verifyPrefixes(rule *FilterRule, prefixes []string) (hasPrefix bool) {
 	if strings.HasPrefix(rule.Element, utils.DynamicDataPrefix) {
-		for _, prefix := range prefixes {
-			if strings.HasPrefix(rule.Element, prefix) {
-				hasPrefix = true
-				break
-			}
-		}
-		if !hasPrefix {
-			return false
+		if hasPrefix = checkPrefix(rule.Element, prefixes); !hasPrefix {
+			return
 		}
 	}
 	for _, value := range rule.Values {
 		hasPrefix = false // reset hasPrefix
 		if strings.HasPrefix(value, utils.DynamicDataPrefix) {
-			for _, prefix := range prefixes {
-				if strings.HasPrefix(value, prefix) {
-					hasPrefix = true
-					break
-				}
-			}
-			if !hasPrefix {
-				return false
+			if hasPrefix = checkPrefix(value, prefixes); !hasPrefix {
+				return
 			}
 		}
 	}
 	return true
 }
 
-//PartialPass is almost the same as Pass except that it verify if the
+//LazyPass is almost the same as Pass except that it verify if the
```

Comment:
Please correct the description here since it does not reflect the functionality.